src/ResourceManager/Insights/Commands.Insights/Autoscale/RemoveAzureRmAutoscaleSettingCommand.cs

```diff
@@ -32,7 +32,7 @@ public class RemoveAzureRmAutoscaleSettingCommand : ManagementCmdletBase
         /// </summary>
         [Parameter(ParameterSetName = RemoveAzureRmAutoscaleSettingParamGroup, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The resource group name")]
         [ValidateNotNullOrEmpty]
-        public string ResourceGroup { get; set; }
```

Comment:
@gucalder same comment about adding an alias to `ResourceGroup`

Reply:
Done